lib/player.js

```diff
@@ -2406,6 +2421,33 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       this.loadEventManager_.listen(
           mediaElement, 'timeupdate', onVideoProgress);
       this.onVideoProgress_();
+      if (this.manifest_.nextUrl) {
+        if (this.config_.streaming.minCompletionPreloadNextUrl > 0 &&
+            this.config_.streaming.minCompletionPreloadNextUrl < 100) {
+          const onTimeUpdate = async () => {
+            const completionRatio =
+                this.video_.currentTime / this.video_.duration;
+            if (!isNaN(completionRatio)) {
+              const percent = Math.round(100 * completionRatio);
+              const minCompletionPreloadNextUrl =
```

Comment:
If this var is moved to the outer if on line 2424, the if that checks it's in the 0-100 range will be slightly simplified.

Reply:
Leaving it as is allows hot switching, otherwise it would only take effect when the config is loaded.